src/test/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/CLIRunnerTest.java

```diff
@@ -41,7 +41,7 @@ public void perform_overrideTokenFromAdditionalArgs_useInvalidToken_executionSho
         Assert.assertFalse("Execution should return 'false' when providing invalid token", successfulExecution);
     }
 
-    @Test(expected = SeaLightsIllegalStateException.class)
+//    @Test(expected = SeaLightsIllegalStateException.class)
```

Comment:
Why did you comment it out?

Reply:
This should be deleted since it tests the logic of throw up sealights exceptions.
Fixed.